lms/djangoapps/verify_student/models.py

```diff
@@ -1215,3 +1215,13 @@ class VerificationAttempt(TimeStampedModel):
         blank=True,
         db_index=True,
     )
+
+    @property
+    def expiration_datetime(self):
+        """Backwards compatibility with existing IDVerification models"""
+        return self.expiration_date
```

Comment:
Do you want me to rename my field `expiration_datetime`, just to limit the number of these backwards compatibility properties we need? It's a more accurate name anyway.

Reply:
I think that's a good idea

Reply:
I'll do that tomorrow morning and ping you for a rebase.